.github/workflows/ci.yml

```diff
@@ -35,9 +35,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - { sbt: 1_0_0, java: 8 }
```

Comment:
Did you remove this because the plugin no longer works on sbt 1.0.0?
I'm not sure how important it is to support such old sbt versions, but it would be nice to keep it unless it gets too complicated.